js/everything.js

```diff
@@ -402,6 +410,28 @@ function loadTime() {
     }
 }
 
+// Key to the persisted volume level entry in localStorage
+var LOCAL_STORAGE_VOLUME_KEY = "VOLUME_LEVEL";
+
+// The delay to debounce writes of the volume level to localStorage
+var VOLUME_PERSIST_DELAY = 200;
+
+// The timeout ID tracking debounced writes of volume level to localStorage
+var debouncedPersistVolumeTimer;
+
+function storeVolume(volume) {
+    if (debouncedPersistVolumeTimer) {
```

Comment:
It would be up to you (though moot given the comments below), but FYI `localStorage.setItem()` is a blocking disk write. The size of the write here is tiny, so it's unlikely to result in noticeable UI stutter in practice; I think the more relevant concern for an application like this would be minimizing energy consumption for laptop users.